agenthub/codeact_agent/system_prompt.j2

```diff
@@ -28,7 +28,7 @@ IMPORTANT:
 - Indentation is important and code that is not indented correctly will fail and require fixing before it can be run.
 - Any code issued should be less than 50 lines to avoid context being cut off!
 - After EVERY `create_file` the method `append_file` shall be used to write the FIRST content!
-- For `edit_file_by_replace` NEVER provide empty parameters!
```

Comment:
New prompt changes after the evaluation started 😩 - Can we hold off on these prompt changes to a separate PR so we can ensure the number I'm about to finish evaluating accurately reflects the prompt for this PR?

Reply:
Doh! I didn't think of that, sorry! Will revert it asap! 😔

Reply:
No worries at all! 😄  just making sure our number accurately reflect the state of this PR